core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeRelationBuilder.scala

```diff
@@ -7,6 +7,25 @@ import scala.collection.JavaConverters._
 
 class SnowflakeRelationBuilder extends SnowflakeParserBaseVisitor[ir.Relation] {
 
+  override def visitSelect_statement(ctx: Select_statementContext): ir.Relation = {
```

Comment:
Moved this method here from `SnowflakeAstBuilder` (as I need it to return a `Relation` instead of a mere `TreeNode`).